example-configs/templates.conf

```diff
@@ -265,13 +265,13 @@ TEMPLATE_LABEL += stretch+standard:debian-9
 
 # Location of templates flavors that are not in default location.
 # Example: wheezy+whonix-gateway would normally be in
+#          $$$$SCRIPTSDIR/wheezy+whonix-gateway
+#   -or-   $$$$SCRIPTSDIR/wheezy
 #          (Don't Place in {curly} brackets; ending curly gets cut off
-#          $$SCRIPTSDIR/wheezy+whonix-gateway
-#   -or-   $$SCRIPTSDIR/wheezy
 TEMPLATE_FLAVOR_DIR :=
-TEMPLATE_FLAVOR_DIR += +gnome:$$SCRIPTSDIR/gnome
-TEMPLATE_FLAVOR_DIR += +flash:$$SCRIPTSDIR/flash
-TEMPLATE_FLAVOR_DIR += +desktop:$$SCRIPTSDIR/desktop
+TEMPLATE_FLAVOR_DIR += +gnome:$$$$SCRIPTSDIR/gnome
+TEMPLATE_FLAVOR_DIR += +flash:$$$$SCRIPTSDIR/flash
+TEMPLATE_FLAVOR_DIR += +desktop:$$$$SCRIPTSDIR/desktop
```

Comment:
This is ugly...
Can we think of some better API for that? For example if the path here is relative, it is based on `$TEMPLATE_SCRIPTS`/`$SCRIPTSDIR`?

Reply:
Ugly yes, although syntactically correct. (Is it because of expansion here and in template-builder make, like secondary expansion?)
I've commented elsewhere on the rationale: do we want to tie $TEMPLATE_SCRIPTS and $SCRIPTSDIR?